third-party/crowdsec/sample-configuration/collections/james.yaml

```diff
@@ -2,5 +2,5 @@ parsers:
   - crowdsecurity/syslog-logs
   - linagora/james-connect-security-log
 scenarios:
-  - crowdsecurity/james
+  - linagora/james-bf-auth
```

Comment:
`linagora` ? are u sure?
I don't see this directory in pr

Reply:
Likely he changed the scenario name:
https://github.com/apache/james-project/pull/1759/files#diff-f1c760ac835c1f664da5ab323a5b6681b1dee8c7c2b0cbd740d2572860193568R2

Reply:
Actually I missed that but I agree with @vttranlina 
@thanhbv200585 can you change back to sth like you did before, crowdsecurity is fine. Remind that James repo is not the property of Linagora :)

Reply:
Well, I see that Crowdsec allows that someone can set their own name for their contributing collections or parsers. crowdsecurity name only for those written by crowdsecurity.
 
![image](https://github.com/apache/james-project/assets/87489927/b3639a42-7743-40ea-a862-f747cca5f7cd)

Reply:
> Actually I missed that but I agree with @vttranlina
@thanhbv200585 can you change back to sth like you did before, crowdsecurity is fine. Remind that James repo is not the property of Linagora :)

I see what you mean. But what if this collection belongs to Crowdsec? When this Linagora collection contributed into Crowdsec Hub?

Reply:
Ok then put apache/james-bf-auth then :)

Reply:
maybe use a full name like `brute-force` to easier understand too?

Reply:
> Ok then put apache/james-bf-auth then :)

Actually after relooking your screenshot... `apache-james` would be better, cause there seems to be other apache products :)

Reply:
@quantranhong1999  Well, I think we should set name `bf` since most collections, scenarios, parsers in Crowdsec Hub use `bf`
![image](https://github.com/apache/james-project/assets/87489927/50ba0ad7-9caa-4cf9-880e-7d4b357b9146)

Reply:
@Arsnael Yep, we can put `apache/james-bf-auth` to minimize controversial in community

Reply:
I think `apache-james/bf-auth` to be better :)